lib/state/appActions.js

```diff
@@ -68,6 +68,10 @@ appActions.errorStages = {
 };
 
 function extractMessage(err) {
+  // if there's no message, just try to stringify the whole obj
+  if (!err.message && typeof err.error === 'string') {
```

Comment:
is there a reason why not continuing on with the if else flow you have below??

also do you think this reads a bit better ?

``if (_.isEmpty(err.message) && typeof err.error === 'string' ) { ...``